### PR TITLE
Add org.gnu.liquidwar6

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://savannah.gnu.org/bugs/index.php?65655
- [ ] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.gnu.liquidwar6

**Comment by Mailaender** [comment]
bot, build org.gnu.liquidwar6

**Comment by hfiguiere** [comment]
This wasn't done

> Please describe your application briefly.

File: org.gnu.liquidwar6.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      - type: shell
+        commands:
+          - autoreconf -fi
+
+  - name: csound
+    buildsystem: cmake
+    build-options:
+      env:
+        PYTHONHOME: /app
+        PYTHONPATH: /usr/lib/python3.10
```
Comment (hfiguiere) [inline org.gnu.liquidwar6.yaml:108]: This runtime has python 3.11

File: org.gnu.liquidwar6.yaml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      arch:
+        arm:
```
Comment (hfiguiere) [inline org.gnu.liquidwar6.yaml:110]: `arm` hasn't bee supported for years. You can remove this.

File: org.gnu.liquidwar6.yaml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      - install -Dm644 org.gnu.liquidwar6.metainfo.xml /app/share/metainfo/org.gnu.liquidwar6.metainfo.xml
+    sources:
+      - type: archive
+        url: http://ftp.gnu.org/gnu/liquidwar6/liquidwar6-0.6.3902.tar.gz
+        sha256: d605549afae403439524e572322e45ebf5b6db8ea59505e5d51cb581a8b5e6a4
+      - type: shell
+        commands:
+          - autoreconf -ivf
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/liquidwar6/0.6.3902/liquidwar6-extra-maps-0.6.3902.zip
```
Comment (hfiguiere) [inline org.gnu.liquidwar6.yaml:139]: are you should this doesn't need to be extracted in a separate directory inside the build tree?

**Comment by Mailaender** [comment]
> Please describe your application briefly.

Of course it was

```xml
  <summary>Quick tactics game</summary>
...
  <description>
    <ul>
      <li>Fast-paced tactical action. Control the movement of all your forces at the same time</li>
      <li>Simple yet original game mechanics, discovered by Thomas Colcombet</li>
      <li>10 different teams, each with their own charactaristics and special attacks</li>
      <li>13 levels, each one twisting the gameplay in a new direction</li>
      <li>100+ more levels if you download the bonus pack</li>
      <li>Levels can be created using simple image files</li>
      <li>Lots of level design options: walls, edge wraparound, bridges, different speed, special areas such as glue or danger and varied army size and fighter behavior</li>
    </ul>
  </description>
```

**Comment by Mailaender** [comment]
I don't understand why it builds locally but not on the CI/CD servers.

**Comment by hfiguiere** [comment]
> > Please describe your application briefly.
> 
> Of course it was

Of course it wasn't answered, albeit you ticked it.

**Comment by hfiguiere** [comment]
```
        "E: org.gnu.liquidwar6:7: developer-id-missing"
```

that's an appstream validation problem.

**Comment by Mailaender** [comment]
bot, build org.gnu.liquidwar6

File: org.gnu.liquidwar6.yaml (hunk 3 above)
Comment (Mailaender) [reply org.gnu.liquidwar6.yaml]: You are right. This was bogus. Well spotted.

**Comment by Mailaender** [comment]
> ```
>         "E: org.gnu.liquidwar6:7: developer-id-missing"
> ```
> 
> that's an appstream validation problem.

Seems to be a linking problem that I can't reproduce locally

> /usr/lib/gcc/x86_64-unknown-linux-gnu/13.2.0/../../../../x86_64-unknown-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `Dtlist'

> flatpak-builder: Cleanup graphviz Error: module graphviz: Child process exited with code 2

**Comment by Mailaender** [comment]
bot, build org.gnu.liquidwar6

**Pull request closed without merging**